### DISCUSSION TITLE
How do I exclude MediaItem from gatsby-source-wordpress?

### DISCUSSION THREAD

**Discussion by danny-wood** [gatsbyjs/gatsby#34802] [Help]
In my gatsby-source-wordpress config I'm excluding types that I don't need, one of them is MediaItem. If I exclude this type I get the following error: "type.MediaItem.exclude" is not allowed.

How do I exclude MediaItems when running a build?

gatsby-source-wordpress: version 6.7.0

```javascript
{
  resolve: `gatsby-source-wordpress`,
  options: {
    type: {
      MediaItem: {
        exclude: true,
      },
    },
  },
}
```

Thanks

**Comment by portent-andy** [comment]
Is there any update on this? I am also getting this "error" even though it should be perfectly usable according to the documentation.
